docker/wiregrid_actuator/Dockerfile

```diff
@@ -2,23 +2,25 @@ FROM socs:latest
 
 WORKDIR /home/ocs/
 
+# Galil installation information:
+# -https://www.galil.com/sw/pub/all/doc/global/install/linux/ubuntu/
+# -https://www.galil.com/sw/pub/all/doc/gclib/html/python.html
 RUN apt-get update \
  && apt-get install -y apt-utils \
  && apt-get install -y wget libavahi-common-dev libavahi-client-dev libavahi-core-dev libcap-dev libdaemon-dev avahi-daemon \
- && wget http://www.galil.com/sw/pub/all/crypto/GALIL-GPG-KEY-E29D0E4B.asc \
- && apt-key add GALIL-GPG-KEY-E29D0E4B.asc \
- && wget http://www.galil.com/sw/pub/ubuntu/20.04/galil.list \
- && cp galil.list /etc/apt/sources.list.d \
+ && wget https://www.galil.com/sw/pub/apt/all/galil-release_1_all.deb \
+ && apt-get install -y ./galil-release_1_all.deb \
+ && rm ./galil-release_1_all.deb \
  && apt-get update \
- && apt-get install -y --download-only gclib \
- && dpkg -x $(find / -name "gclib*.deb") / \
- && apt-get install -y gclib \
+ && apt install -y gclib \
  && apt-get install -y --download-only gcapsd \
  && dpkg -x $(find / -name "gcapsd*.deb") /tmp/gcapsd/ \
- && mv /tmp/gcapsd/usr/sbin/gcapsd /usr/sbin/ \
+ && mv -v /tmp/gcapsd/usr/sbin/gcapsd /usr/sbin/ \
  && rm -r /tmp/gcapsd/ \
- && tar -xvf /usr/share/doc/gclib/src/gclib_python.tar.gz \
- && python3 setup.py install
+ && cp /usr/share/gclib/src/wrappers/python/* . \
+ && cp /usr/share/gclib/doc/examples/python/* . \
+ && python3 setup.py install \
```

Comment:
I know it's in the upstream instructions, but direct invocation of `setup.py` like this is deprecated. Can we change this line to `python3 -m pip install . \` instead?

I was able to build successfully using this.

Previous build:
```
/opt/venv/lib/python3.10/site-packages/setuptools/_distutils/cmd.py:66: SetuptoolsDeprecationWa
rning: setup.py install is deprecated.
!!

        ********************************************************************************
        Please avoid running ``setup.py`` directly.
        Instead, use pypa/build, pypa/installer or other
        standards-based tools.

        See https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html for details.
        ********************************************************************************

!!
  self.initialize_options()
/opt/venv/lib/python3.10/site-packages/setuptools/_distutils/cmd.py:66: EasyInstallDeprecationW
arning: easy_install command is deprecated.
!!

        ********************************************************************************
        Please avoid running ``setup.py`` and ``easy_install``.
        Instead, use pypa/build, pypa/installer or other
        standards-based tools.

        See https://github.com/pypa/setuptools/issues/917 for details.
        ********************************************************************************

!!
  self.initialize_options()
ERROR setuptools_scm._file_finders.git listing git files failed - pretending there aren't any
zip_safe flag not set; analyzing archive contents...
gclib location: /home/ocs/gclib.py
```

New build:
```
Processing /home/ocs
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Building wheels for collected packages: gclib
  Building wheel for gclib (setup.py): started
  Building wheel for gclib (setup.py): finished with status 'done'
  Created wheel for gclib: filename=gclib-1.0-py3-none-any.whl size=5509 sha256=a5dd87761922a01
d76cf616ca1e2440c5f2f8d6ac9ffddaacc0ca0f68082bc6e
  Stored in directory: /tmp/pip-ephem-wheel-cache-mo6q0wb9/wheels/4a/c9/f4/70900c0ce2d621b50abb
09a7fa008c78c8e367e7453fd3a63c
Successfully built gclib
Installing collected packages: gclib
Successfully installed gclib-1.0
gclib location: /home/ocs/gclib.py
```